torch/csrc/Module.cpp

```diff
@@ -1560,6 +1560,15 @@ Call this whenever a new thread is created in order to propagate values from
     return at::globalContext().linalgPreferredBackend();
   });
 
+  py_module.def(
```

Comment:
For testing, I bet there is an already existing way to go from data_ptr to tensor.

Reply:
Not to my knowledge.. if you have one link me ! For one, data pointer is not a python exposed concept.